### DISCUSSION TITLE
Add GigaCalories to energy

### DISCUSSION THREAD

**Discussion by DevRedOWL** [home-assistant/architecture#1018] [Entity Models] [answered]
Good day, everyone!

In countries of [CIS](https://en.wikipedia.org/wiki/Commonwealth_of_Independent_States), heat consumption are measured in GCal (which are 4,184 GJ)
So, my idea is add this to omit conversions and make native support for energy usage dashboard also

PR: https://github.com/home-assistant/core/pull/105775

**Comment by frenck** [comment]
Which integrations can use this today?

**Reply by DevRedOWL** [reply]
I think by all integrations which uses heat sensor.
Probably landisgyr

**Reply by DevRedOWL** [reply]
@frenck, well, what should i do? I mean is it going to be considered?

**Comment by frenck** [comment, marked as answer]
We have discussed this proposal and are accepting it.

Please open up the pull requests to add this unit.

Thanks! 👍 

../Frenck
